app/resources/start.html.template

```diff
@@ -55,36 +55,39 @@
         </div>
         <div class="end-multi-columns"></div>
       </div>
-      <div class="row stats">
-        {% if num_people %}
-          {% blocktrans %}Currently tracking about {{num_people}} records.{% endblocktrans %}
-        {% endif %}
-      </div>
-      <div class="row custom-message">
-        {{env.start_page_custom_html|default:""|safe}}
+      <div class="row">
```

Comment:
Do you have reason to break "row" div and "stats" div?

Reply:
Yes, it was creating problems in getting desired results. Probably, this was the way how I could fix it

Reply:
I found that "custom-message" class is used in other pages too (you can grep "custom-message" in app/resources/*.html.template), but I guess we don't want to centerize it in other places. Can you leave .custom-message CSS rule as is, and instead define another rule specific to the custom message in this page, like this?:
```css
.main .row.custom-message {
  margin: 24px auto;
}
```

Then you should be able to keep the original HTML structure here (i.e., "row stats" div and "row custom-message" div).

Also, I feel text-align: center is not necessary as long as you use "auto" margin to centerize the box.